addons/Stubborn/readme.md

```diff
@@ -0,0 +1,7 @@
+# Stubborn
+A Windower 4 addon to prevent unnecessary "Call for help!"
+
+Stubborn is a light weight replacement to the old CFHProtect addon. 
+
+
+If you need to call for help. Type either //stubborn or //lua c stubborn.

```

Comment:
No need to mention `//lua c stubborn`, that's not an encouraged way to call commands. It's meant to be used as an API.